lib/hooks/create/unique.js

```diff
@@ -34,6 +35,17 @@ module.exports = settings => {
       .where('status', '!=', 'new')
       .then(results => {
         if (results && results.length) {
+          if (type === 'project' && action === 'grant') {
+            const id = results[0].id;
+            return Task.find(id)
```

Comment:
See https://github.com/UKHomeOffice/taskflow/pull/48

```suggestion
            return model.find(id)
```

Reply:
Note: don't do this until that PR has been merged and published.